spacy/lang/en/stop_words.py

```diff
@@ -66,7 +66,10 @@
 whither who whoever whole whom whose why will with within without would
 
 yet you your yours yourself yourselves
-
-'d 'll 'm 're 's 've
 """.split()
 )
+
+for hyphen in ["'", "`", "‘", "´", "’"]:
```

Comment:
Hypen = apostrophe, right?

Also, maybe we should consider leaving out \` and ´? Those could possible produce unexpected results. Not that it matters much, but it could be confusing if a string like \`m is randomly declared a stop word.

And we can leave out the `"'"` as well, unless that's a different unicode character? Otherwise, the first loop would replace `'` with itself?

Reply:
I'll address your first two points, but I think we need to keep `"'"`, otherwise `n't` wouldn't actually be added to the stopwords as it's not in the original list anymore. The replacement is indeed a bit dummy, but the code is compacter this way?

Reply:
Ah, yes, makes sense 🤦‍♀️ 

I guess an alternative would be something like:

```python
contractions = ["n't", "'d", "'ll", "'m", "'re", "'s", "'ve"]
STOP_WORDS.update(contractions)
# and then go on with the for loop
```